pkg/connection/connection.go

```diff
@@ -650,6 +654,30 @@ func (m *Manager) AddBqConnectionFromConfig(connection *config.GoogleCloudPlatfo
 	}
 	m.mutex.Unlock()
 
+	if len(connection.ServiceAccountFile) == 0 && len(connection.ServiceAccountJSON) == 0 {
+		return errors.New("at least one of service_account_file or service_account_json must be provided")
+	}
+
+	if len(connection.ServiceAccountFile) > 0 && connection.ServiceAccountFile != "" {
+		file, err := ioutil.ReadFile(connection.ServiceAccountFile)
+		if err != nil {
+			return errors.New("failed to read service account file")
```

Comment:
would be nice to add file path as well.
something like:
```
failed to read service account file at `~/docs/key.json`
```

Reply:
also, which connection is it? is connection name shown on the error?

Reply:
we already printing connection name, Search for gcp in logs